### PR TITLE
seeder data for dev server

### PR DESCRIPTION
**Pull request by MarkellRichards**
Stock seeder data for wish cards and users.

### CODE REVIEW

**Comment by gnorbsl** [comment]
looks good so far, can you add an agency to the user that is partner, and make sure that the script doesnt run and exists with error if process.env.LOCAL_DEVELOPMENT is false

File: server/db/seeder.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    await WishCard.deleteMany();
+    process.exit();
+  } catch (error) {
+    process.exit(1);
+  }
+};
+if (process.argv[2] === '-d') {
+  destroyData();
+} else {
+  importData();
```
Comment (Enubia) [inline server/db/seeder.js:37]: would also be nice to check for -i, so that we can insert only, a few of us already have data in their local db

**Comment by MarkellRichards** [comment]
> looks good so far, can you add an agency to the user that is partner, and make sure that the script doesnt run and exits with error if process.env.LOCAL_DEVELOPMENT is false

I am looking at the User model. There is no reference to the agency model.

File: server/db/seeder.js (hunk 1 above)
Comment (MarkellRichards) [reply server/db/seeder.js:37]: Got it, will add option

**Comment by Enubia** [comment]
IMO the agency should hold all the information about users and wishcards related to them since it makes the most sence.

Users -> Agency <- Wishcards

maybe we should discuss this

**Comment by gnorbsl** [comment]
The agency model holds a userId in accountmanager @MarkellRichards , thats what I meant

**Comment by Enubia** [comment]
@MarkellRichards I would take the models and fill all the fields that are available, copy them like 20 times and change some of the data, 3 samples per collection aren't really that much for testing

**Comment by Enubia** [comment]
@MarkellRichards are you done with the seeder data now? or is there anything you want to add to it?

**Comment by MarkellRichards** [comment]
I just need to add more sample data. In the middle of traveling so I won't have internet till tomorrow morning.

**Pull request closed without merging**